pkgdiff.yaml

```diff
@@ -0,0 +1,73 @@
+package:
+  name: pkgdiff
+  version: 1.8
+  epoch: 0
+  description: A tool for visualizing changes in Linux software packages
+  copyright:
+    - license: GPL-2.0
+  dependencies:
+    runtime:
+      - libmagic
+      - perl
+      - wdiff
+
+environment:
+  contents:
+    packages:
+      - bash
+      - binutils
+      - build-base
+      - busybox
+      - file
+      - libmagic-dev
+      - perl-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/lvc/pkgdiff.git
+      expected-commit: 56af1b018404f6efcf05063c1db5490e77bf6b00
+      tag: ${{package.version}}
+
+  - runs: |
+      # create dir structure expected by make-install
+      mkdir -p "${{targets.destdir}}"/usr/bin
+      mkdir -p "${{targets.destdir}}"/usr/share
+
+  - uses: autoconf/make-install
+    with:
+      opts: |
+        PREFIX=${{targets.destdir}}/usr
+
+  - runs: |
+      PERL_MM_USE_DEFAULT=1 perl -MCPAN -e 'install File::LibMagic' # NOTE: running with 'notest' doesn't install the module correctly for some reason
+      mkdir -p ${{targets.destdir}}/usr/local/lib
+      cp -r /usr/local/lib/perl5 ${{targets.destdir}}/usr/local/lib
```

Comment:
This would be better as a separate package, e.g. `perl-file-libmagic`. We've others like `perl-file-fcntllock.yaml` that also show the GPL 1.0 or Artistic license that the package uses as well.

Once that's a separate package, it can be a runtime dependency of the main pkgdiff package.

Reply:
Cool.  I'll tackle that mañana